object-manipulation-1/src/to-object.js

```diff
@@ -0,0 +1,10 @@
+/* exported toObject */
+
+function toObject(keyValuePair) {
+  var newObj = {};
+  for (var i = 0; i < keyValuePair.length; i += 2) {
```

Comment:
This works great, but you can do it without a loop too!
```javascript
newObj[keyValuePair[0]] = keyValuePair[1]
```

Reply:
Oh, nice. Thanks!